pkgs/tools/misc/diffoscope/default.nix

```
@@ -35,7 +35,7 @@ python3Packages.buildPythonApplication rec {
   #
   # Still missing these tools: abootimg docx2txt dumpxsb enjarify js-beautify lipo oggDump otool procyon-decompiler Rscript wasm2wat zipnode
   # Also these libraries: python3-guestfs
-  pythonPath = with python3Packages; [ debian libarchive-c python_magic tlsh rpm ] ++ [
+  pythonPath = with python3Packages; [ debian libarchive-c python_magic tlsh rpm xattr ] ++ [
```

I think it should be [`pyxattr`](https://salsa.debian.org/reproducible-builds/diffoscope/blob/7583af27272fce16d413ad0ca6c79745180f68d3/setup.py#L66)
```suggestion
  pythonPath = with python3Packages; [ debian libarchive-c python_magic tlsh rpm pyxattr ] ++ [
```

---

I'm not quite clear on the difference but I can confirm it works with `pyxattr` as well, updated.